spec/sidekiq.rb

```diff
@@ -0,0 +1,13 @@
+# This is a mock sidekiq module for testing purposes.
+
+module Sidekiq
+  module Worker
```

Comment:
We can add `sidekiq` and `aws-dynamodb` to `Gemfile` so that we can avoid mocking